specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3925,6 +3931,16 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add data disks to the virtual machines in the scale set. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "items": {
+            "$ref": "./computeRPCommon.json#/definitions/diskControllerType"
+          },
+          "x-ms-enum": {
+            "name": "DiskControllerTypes",
+            "modelAsString": true
+          }
```

Comment:
I don't think you need this either unless this modelAsString value is different from the one in computeRPCommon, you only need the ref and the description

Reply:
makes sense. Removing